nix/modules/lanzaboote.nix

```diff
@@ -7,6 +7,12 @@ let
   };
 
   configurationLimit = if cfg.configurationLimit == null then 0 else cfg.configurationLimit;
+  timeout = if config.boot.loader.timeout == null then 0 else config.boot.loader.timeout;
+
+  systemdBootLoaderConfig = pkgs.writeText "loader.conf" ''
+    timeout ${toString timeout}
+    console-mode ${config.boot.loader.systemd-boot.consoleMode}
```

Comment:
Should we set `editor = no` here as well or make it configurable? I assume disabling the editor is a necessary security precaution?

Reply:
My understanding was that systemd-boot forbids editing anything when the machine is in Secure Boot mode. We could add it for good measure though. However, wouldn't it be nice for #58 to have an editor when SB is disabled?

Reply:
I think, editor can be used, but it won't have any effect. I would argue this is out of scope for this PR.

Reply:
I'll leave it as is. If it every turns out to be a problem, we can solve it in a follow up PR.